app/modules/new-vm/view.jade

```diff
@@ -154,6 +154,16 @@ form.form-horizontal(ng-submit="createVM()")
                   i.fa.fa-plus
                   |  Add interface
     //- end of misc and interface panel
+  //- Cloud config panel
+  .grid(ng-if = 'cloudConfig')
+    .panel.panel-default
+      .panel-heading.panel-title
+        i.fa.fa-cloud
+        |  Cloud config
+      .panel-body
+        textarea.form-control(rows="10")
+          | {{cloudConfig}}
+
```

Comment:
TODO: fold the default config and unfold only for edition. Then save it for passing it to xo-server